Dockerfile

```diff
@@ -1,33 +1,16 @@
-FROM bitnami/minideb:jessie
-
-MAINTAINER David Barranco <d.baus@outlook.com>
```

Comment:
As the MAINTAINER label it's deprecated, you could set:

```
LABEL maintainer="arp4@protonmail.com"
```

:D 
The rest LGTM!